app/gateway/database/postgres/pgtest/docker.go

```diff
@@ -0,0 +1,64 @@
+package pgtest
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/ory/dockertest"
+)
+
+var testPool *pgxpool.Pool
+
+func SetupTests() (teardownFn func(), err error) {
+	dbName := GetRandomDBName()
+	pool, err := dockertest.NewPool("")
+	ctx := context.Background()
+	if err != nil {
+		log.Fatalf("error when trying to connect to docker")
```

Comment:
Aqui precisa de um return, pois do contrário a execução continuará.

Reply:
Xenia, nesse caso como eu to usando o log.Fatalf, ele não iria parar a execução completamente?
Ao menos pelo que eu li da função, ele já da o log da mensagem, e chamada o os.Exit(), parando completamente o teste.